File: Ex5344.java
import java.util.Locale;
import java.util.Scanner;

public class Ex5344 {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);
        int countPares = 0;

        int n = sc.nextInt(); // botas entregues
        for (int i = 0; i < n; i++) {
            String input = sc.nextLine().replace(" ", "");

            String[] pes = new String[61];

            pes[i] = input;


        }

        System.out.println(countPares);
    }
}

File: ex5344.py
n = int(input())
from collections import Counter
botas = []
pares = 0
ant = ""

for i in range(n):
    bota = input()
    botas.append(bota.replace(" ", ""))

a = Counter(botas)

for q in a.values():
    if q > 1:
        pares+=1

print(pares)
    